eland/dataframe.py

```diff
@@ -1386,9 +1386,12 @@ def aggregate(self, func, axis=0, *args, **kwargs):
         # ['count', 'mad', 'max', 'mean', 'median', 'min', 'mode', 'quantile',
         # 'rank', 'sem', 'skew', 'sum', 'std', 'var', 'nunique']
         if isinstance(func, str):
-            # wrap in list
-            func = [func]
-            return self._query_compiler.aggs(func)
+            if hasattr(self, func):
```

Comment:
I think we can do this instead to guarantee that we only run actual aggs like `mean`, etc:

```python
        if isinstance(func, str):
            return self._query_compiler.aggs([func]).squeeze().rename(None)
```

Reply:
@sethmlarson That was a brilliant thought of converting dataframe to series. I was just trying to work whole around and making it complex.

I added one more edge case where ed_df.agg(["mean"]) also returns dataframe to series.

Reply:
Actually our previous behavior here was correct:

```python
>>> print(df.agg("mean"))
FlightTimeMin     511.127842
FlightTimeHour      8.518797
dtype: float64

>>> print(df.agg(["mean"]))
      FlightTimeMin  FlightTimeHour
mean     511.127842        8.518797
```

If there isn't one already there should be a test case for this behavior.